<launch>
    <node name="battery_monitor" pkg="slaw_diagnostics" type="slaw_battery_monitor" output="screen">
    	<param name="serial_port" value="/dev/youbot/lcd_display" />
    </node>

    
    <node pkg="diagnostic_aggregator" type="aggregator_node" name="diagnostic_aggregator" output="screen" >
        <rosparam command="load" file="$(find slaw_diagnostics)/config/slaw_diagnostics.yaml" />
    </node>

    <node name="sysinfo" pkg="slaw_diagnostics" type="sysinfo.py" output="screen" />

</launch>
